package.json

```diff
@@ -64,8 +70,29 @@
     "loadicons": "^1.0.0",
     "lunr": "^2.3.6",
     "markdown-it": "^12.3.2",
+    "markdown-it-anchor": "^8.6.5",
     "npm-run-all": "^4.1.5",
+    "postcss": "^8.4.19",
+    "postcss-calc": "^6.0.0",
+    "postcss-cli": "^10.1.0",
+    "postcss-dir-pseudo-class": "^5.0.0",
+    "postcss-discard-empty": "^4.0.1",
+    "postcss-dropdupedvars": "^1.1.2",
+    "postcss-dropunusedvars": "^1.2.1",
+    "postcss-focus-ring": "^1.0.0",
+    "postcss-functions": "^3.0.0",
+    "postcss-import": "^15.0.0",
+    "postcss-inherit": "^4.0.3",
+    "postcss-logical": "^4.0.2",
+    "postcss-mixins": "^6.2.1",
+    "postcss-nested": "^3.0.0",
+    "postcss-remapvars": "^1.1.0",
+    "postcss-selector-parser": "^6.0.2",
+    "postcss-svg": "^3.0.0",
+    "postcss-values-parser": "^3.1.1",
+    "prettier": "^2.8.0",
```

Comment:
Since we're pulling out the build updates from this PR, I'm guessing we can remove all the postcss dependencies. @pfulton and I also discussed holding off adding prettier for now and adding it as it's own PR.